06/ht/src/AboutInfo.js

```diff
@@ -0,0 +1,9 @@
+function showAboutContent(element) {
+  element.innerHTML = `
+    <div class="content_about">
+        <h2>Информация о себе</h2>
+    </div>
+    `;
+}
+
+module.exports = showAboutContent;
```

Comment:
```suggestion
module.exports = {showAboutContent};
```

и в месте вызова тогда будет

`const {showAboutContent} = require('./aboutInfo')`

Такой подход позволит тебе использовать именно такие же названия функций, которые ты объявил в файле.